streams-bootstrap-large-messages/build.gradle.kts

```diff
@@ -0,0 +1,6 @@
+description = "Utils for using Large Message SerDe with your Kafka Streams Application"
+
+dependencies {
+    api(project(":streams-bootstrap"))
+    implementation(group = "com.bakdata.kafka", name = "large-message-core", version = "2.5.1")
```

Comment:
Could also be `api` given that you most likely need to access the config

Reply:
What do you prefer? That way the user needs to explicitly specify a version of LMS

Reply:
And also this dependency does not include the Serde which the user most likely needs to add